src/components/pages/AddReviews/MentorAddReview.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
useEffect is imported but never used - delete this to make it cleaner

Reply:
Doing so right now!